# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/style/ImageLoader.h
# Commit: 2e365eaea75b
# Full Hash: 2e365eaea75b925b2b5ff621634ba8f7d7fec00b
# Author: Edgar Chen <echen@mozilla.com>
# Date: 2019-08-22 04:04:03
# Regressor Bug: 1568950
# File Overlap Count: 1
# Description:
#   Bug 1568950 - Part 5: Remove nsIDOMWindowUtils::forceUseCounterFlush; r=smaug
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D42818
# ==============================================================================

diff -r 1c1c4ca0bc14 -r 2e365eaea75b layout/style/ImageLoader.h
--- a/layout/style/ImageLoader.h	Wed Aug 21 10:29:14 2019 +0000
+++ b/layout/style/ImageLoader.h	Wed Aug 21 12:52:19 2019 +0000
@@ -84,8 +84,6 @@
   // May be called from any thread.
   static void DeregisterCSSImageFromAllLoaders(const StyleLoadData&);
 
-  void FlushUseCounters();
-
  private:
   // This callback is used to unblock document onload after a reflow
   // triggered from an image load.
